_content/monographs/practical-advice-for-meditators_mills-laurence.md

```diff
@@ -0,0 +1,23 @@
+---
+title: "Practical Advice for Meditators"
+authors:
+  - mills-laurence
+external_url: "https://www.bps.lk/olib/wh/wh116_Khantipalo_Practical-Advice-For-Meditators.pdf"
```

Comment:
Replace the .pdf at the end here with .html  The Drive link below is sufficient for the PDF 😁